source/_posts/2020-02-05-release-105.markdown

```diff
@@ -483,7 +483,7 @@ Hats over your heart for these shuttered integrations. Pour one out for:
 
 - __Hook__ *Removed* - The `hook` integration has been removed due to the shutdown of the connected service. All servers that have been providing this service will be turned off on 31 Jan 2020. - ([@springstan] - [#31046])
 
-## Release 0.105.1 - February 5
+## <a id="release-0.105.1"></a>Release 0.105.1 - February 5
```

Comment:
All release notes already come with an anchor defined by markdown. We've been moving away from trying to do custom markdown. 

It's also generated here fyi https://github.com/home-assistant/hass-release/blob/master/hassrelease/changelog.py#L152

Reply:
The problem with the already existing automatic markdown anchors is that the title for which is generated includes the date, e.g. the "February 5" part above, and this ends up as part of the anchor (`release-01051---february-5` for the above). That date is not available in the UI where the release notes link is generated, all we have there is the version.